OneCommunityGlobal/HighestGoodNetworkApp
# Alan fix subtask folder fix

**This PR fixes WBS Subtask ‘folder’ button and edit bar showing incorrectly**
When having a sub task folder open, close and reopen the mother task’s folder. The sub task folder will remain open but not showing any sub tasks
Also when having a sub task edit bar open, close the mother task’s folder. The edit bar for that subtask won’t disappear
Example on the [video](https://drive.google.com/file/d/1USQPxQk-XefglMeRjmiwMqzycbodkjss/view?usp=sharing)
Here is the video of the problem as well.

https://user-images.githubusercontent.com/67836976/224519149-1aa5c5c6-6562-45e6-9dcd-1c2ba516b302.mp4

How to test:
- Log in as admin.
- Go to projects.
- Choose a project ( I chose AAA USE THIS PROJECT / Jae's WBS - USE THIS ).
- Open a folder, then open a subfolder that has subtasks or subfolders.
- Close the first folder you opened, and then reopen.
- Check if all subfolders are closed.
- Now open another folder.
- Click the Edit button for one of subtasks or subfolders.
- Close the first folder you opened, and check if edit options are gone.
- Reopen the folder and check that you cannot see the edit options.

Please check the video to understand the problem better.
